build/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# Mount the project as a read only filesystem to /app and pass
+# along any arugments that were provided to this script
+docker run -v"$(pwd):/app:ro" backus/rubocop-rspec "$@"
```

Comment:
I'd recommend using `--volume` instead of `-v` in scripts.

I think it's fine, maybe even preferable, when you are working on shell to use the short letter options. In scripts though I think that if a longer option exists it should be preferable. While it is verbose it helps later on when you are trying to understand some function when you can more easily search for it in docs and online.